app/gui/view/component-browser/searcher-list-panel/src/lib.rs

```diff
@@ -0,0 +1,633 @@
+    /// Radius of the rounded corners.
+    corner_radius:    f32,
+    /// Extra space between scroll area and backgound shape edge.
+    padding:          f32,
+    /// Color used for the panel background.
+    background_color: color::Rgba,
+}
+
+#[derive(Clone, Debug, Default)]
+struct SectionHeadingStyle {
+    /// Font size used for section headers.
+    size:   text::style::Size,
+    /// Font used for section headers.
+    font:   String,
+    /// Color used for section headers.
+    color:  color::Rgba,
+    /// Distance between the section header and the section content.
+    offset: f32,
+}
+
+impl SectionHeadingStyle {
+    /// Returns the size of the header, which consists of the header text and the padding between
+    /// text and content.
+    fn height(&self) -> f32 {
+        self.size.raw + self.offset
+    }
+}
+
+#[derive(Clone, Debug, Default)]
+struct SectionStyle {
+    heading: SectionHeadingStyle,
+    /// Thickness of the line that divides the sections within the Component List Panel.
+    divider_height: f32,
+    /// Color used for the Divider.
+    divider_color: color::Rgba,
+    favourites_section_base_color: color::Rgba,
+}
+
+#[derive(Clone, Debug, Default)]
+struct MenuStyle {
+    /// Height of the area reserved for Component Browser Panel Menu.
+    height:         f32,
+    /// Thickness of the line that divides the Component List Panel from the Component Browser
+    /// Panel Menu.
+    divider_height: f32,
+    /// Color used for the Divider.
+    divider_color:  color::Rgba,
+}
+
+impl Style {
+    fn from_theme(
+        network: &enso_frp::Network,
+        style: &StyleWatchFrp,
+    ) -> (enso_frp::Stream<Style>, enso_frp::stream::WeakNode<enso_frp::SourceData>) {
+        let theme_path: style::Path = list_panel_theme::HERE.into();
+
+        let content_width = style.get_number(theme_path.sub("content_width"));
+        let content_height = style.get_number(theme_path.sub("content_height"));
+
+        let content_padding = style.get_number(theme_path.sub("content_padding"));
+
+        let menu_height = style.get_number(theme_path.sub("menu_height"));
+        let menu_divider_color = style.get_color(theme_path.sub("menu_divider_color"));
+        let menu_divider_height = style.get_number(theme_path.sub("menu_divider_height"));
+
+        let content_corner_radius = style.get_number(theme_path.sub("content_corner_radius"));
+
+        let section_divider_height = style.get_number(theme_path.sub("section_divider_height"));
+
+        let section_heading_size = style.get_number(theme_path.sub("section_heading_size"));
+        let section_heading_font = style.get_text(theme_path.sub("section_heading_font"));
+        let section_heading_color = style.get_color(theme_path.sub("section_heading_color"));
+        let section_divider_color = style.get_color(theme_path.sub("section_divider_color"));
+
+        let content_background_color = style.get_color(theme_path.sub("content_background_color"));
```

Comment:
Maybe there is some reason for that, I just don't see it yet - why this spacing between lines? Looks random to me.